docs/releases.md

```diff
@@ -0,0 +1,127 @@
+
+# Release Documentation
+
+# Abstract
+
+In order to provide a generic guidance to users, downstream maintainers and
+any other consumers of the Cloud Hypervisor project, this document
+describes the release process, release cadence, stability expectations and
+related topics.
+
+# Basic Terms
+
+## Stability
+
+In the context of Cloud Hypervisor, the points of reference regarding stability
+are as below:
+
+- [REST API](api.md#rest-api).
+- [Command line options](api.md#command-line-interface).
+- [Device Model](device_model.md).
```

Comment:
This might need reviewing. It's probably out of date.

Reply:
For this one, did you mean this point is not yet to be included? Otherwise, seems device model might be the subject of a separate PR.

Thanks